### PR TITLE
[FORMS-2938] made responsivegrid as the default layout for panelconta…

### PR DESCRIPTION
**Pull request by deepprakash345**

### CODE REVIEW

**Review by nit23uec — APPROVED** [review]

**Comment by deepprakash345** [comment]
Can we merge this as the CI has also passed
cc:- @nit23uec , @sakshi-arora1

File: examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml [hunk 1]
```diff
@@ -159,6 +161,19 @@
                                 jcr:primaryType="nt:unstructured"/>
                     </default>
                 </datepicker>
+                <panelcontainer jcr:primaryType="nt:unstructured">
+                    <default
+                            jcr:lastModified="{Date}2023-03-07T11:41:57.902+05:30"
+                            jcr:lastModifiedBy="admin"
```
Comment (sakshi-arora1) [inline examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml:167]: pleaseremove admin

File: examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
             <page jcr:primaryType="nt:unstructured">
                 <policy_212634895490169
-                    jcr:primaryType="nt:unstructured"
-                    jcr:title="Core Components Examples - Page"
-                    sling:resourceType="wcm/core/components/policy/policy"
-                    ampMode="pairedAmp"
-                    appResourcesClientlib="cmp-examples.site"
-                    clientlibs="[cmp-examples.base,forms-examples.base]">
+                        jcr:primaryType="nt:unstructured"
+                        jcr:title="Core Components Examples - Page"
```
Comment (sakshi-arora1) [inline examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml:11]: Form Core Components Examples...

**Review by sakshi-arora1 — COMMENTED** [review]
Minor comments. Rest LGTM

File: examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml (hunk 2 above)
Comment (deepprakash345) [reply examples/ui.content/src/main/content/jcr_root/conf/core-components-examples/settings/wcm/policies/.content.xml]: done.

**Comment by deepprakash345** [comment]
> Minor comments. Rest LGTM

made the suggested changes

**Pull request merged**